winui/Kanban/Workflows.md

```diff
@@ -0,0 +1,317 @@
+            taskDetail.Id = "29477";
+            taskDetail.Description = "Null reference exception thrown in line chart";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "Normal";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle10.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "WPF Issue";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Minimum and maximum properties are not working in dynamic update";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "Bug Fixing" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle11.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "Kanban Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Need to implement tooltip support for Kanban";
+            taskDetail.Category = "Review";
+            taskDetail.IndicatorColorKey = "High";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle12.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Dragging events support for Kanban";
+            taskDetail.Category = "Closed";
+            taskDetail.IndicatorColorKey = "Normal";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle13.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+
+            taskDetail = new KanbanModel();
+            taskDetail.Title = "New Feature";
+            taskDetail.Id = "29574";
+            taskDetail.Description = "Swimlane support for Kanban";
+            taskDetail.Category = "Open";
+            taskDetail.IndicatorColorKey = "Low";
+            taskDetail.Tags = new List<string>() { "New Control" };
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle14.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+        #endregion
+    }
+
+{% endhighlight %}
+{% endtabs %}
```

Comment:
@Vasanthsf4654 , Add these md file in toc file

Reference - https://github.com/syncfusion-content/winui-docs/blob/development/winui-toc.html#L381